ronn/geopm_report.7.ronn

```diff
@@ -0,0 +1,288 @@
+                    the sticker (base) frequency.
+
+  `time-hint-network (s)`: The portion of sync-runtime where the
+                           region hint was GEOPM_REGION_HINT_NETWORK.
+                           The region hint is determined by the hint
+                           passes to `geopm_region()` for the region
+                           and any nested regions.
+
+  `time-hint-ignore (s)`: The portion of sync-runtime where the region
+                          hint was GEOPM_REGION_HINT_IGNORE.  The
+                          region hint is determined by the hint passes
+                          to `geopm_region()` for the region and any
+                          nested regions.
+
+  `time-hint-compute (s)`: The portion of sync-runtime where the
+                           region hint was GEOPM_REGION_HINT_COMPUTE.
+                           The region hint is determined by the hint
+                           passes to `geopm_region()` for the region
+                           and any nested regions.
+
+  `time-hint-memory (s)`: The portion of sync-runtime where the region
+                          hint was GEOPM_REGION_HINT_MEMORY.  The
+                          region hint is determined by the hint passes
+                          to `geopm_region()` for the region and any
+                          nested regions.
+
+  `time-hint-io (s)`: The portion of sync-runtime where the region
+                      hint was GEOPM_REGION_HINT_IO.  The region hint
+                      is determined by the hint passes to
+                      `geopm_region()` for the region and any nested
+                      regions.
+
+  `time-hint-serial (s)`: The portion of sync-runtime where the region
+                          hint was GEOPM_REGION_HINT_SERIAL.  The
+                          region hint is determined by the hint passes
+                          to `geopm_region()` for the region and any
+                          nested regions.
+
+  `time-hint-parallel (s)`: The portion of sync-runtime where the
+                            region hint was
+                            GEOPM_REGION_HINT_PARALLEL.  The region
+                            hint is determined by the hint passes to
+                            `geopm_region()` for the region and any
+                            nested regions.
+
+  `time-hint-unknown (s)`: The portion of sync-runtime where the
+                           region hint was GEOPM_REGION_HINT_UNKNOWN.
+                           The region hint is determined by the hint
+                           passes to `geopm_region()` for the region
+                           and any nested regions.
+
+  `time-hint-unset (s)`: The portion of sync-runtime where the region
+                         hint was GEOPM_REGION_HINT_UNSET.  The region
+                         hint is unset when exiting a region
+                         (i.e. when a CPU is in the unmarked region).
+
```

Comment:
Maybe a bit more explicit here: I time-hint-unset only possible when geopm controller considers the app to be in the unmarked region? Why not call this something like time-hint-unmarkedregion then (less confusing)?

Reply:
It is also possible for a user to create a region with GEOPM_REGION_HINT_UNSET if they really want to.